config/m4/compiler.m4

```diff
@@ -598,6 +598,10 @@ ADD_COMPILER_FLAGS_IF_SUPPORTED([[-Wno-pointer-sign],
                                 [AC_LANG_SOURCE([[int main(int argc, char **argv){return 0;}]])])
 
 
+# Prevent libtool from suppression of warnings
+LT_CFLAGS="-no-suppress"
+AC_SUBST(LT_CFLAGS)
```

Comment:
move this after line 605, `AC_SUBST([BASE_CFLAGS])`

Reply:
done